math/benches/criterion_field.rs

```diff
@@ -25,6 +36,32 @@ pub fn starkfield_ops_benchmarks(c: &mut Criterion) {
         bench.iter(|| x * y);
     });
 
+    group.bench_with_input("pow by 1", &x.clone(), |bench, x| {
+        bench.iter(|| x.pow(1_u64));
+    });
```

Comment:
For this and the `square with pow` cases I would add a comment clarifying the non-boxed constant is intentional as it is how it would be used in the wild for their particular use cases.